_includes/footer.html

```diff
@@ -51,8 +51,11 @@ <h1 class="title">Get Involved</h1>
                 <a href="https://www.grin-forum.org/"><i class="fab fa-discourse"></i>Forum</a>
               </li>
               <li>
-                <a href="https://discord.gg/Z3sEfEU"><i class="fab fa-discord"></i>Discord</a>
+                <a href="https://keybase.io/team/grincoin"><i class="fab fa-keybase"></i>Keybase chat & trade</a>
```

Comment:
Maybe just "Keybase"? I think we should be making it more prominent that keybase should be used for trading but I don't think this is the right place.